capstone/backend/src/main/java/com/google/servlets/UserPreferences.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2020 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.servlets;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class UserPreferences {
```

Comment:
Can you use AutoValue to create this class?

Reply:
I tried using AutoValue but it doesn't support  an empty no-arg constructor. I need this no-arg constructor for to use gson to convert a json strong to a UserPreferences object.